### PR TITLE
refactored api/server.js tests

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

[description]

medic/cht-core#[number]

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -230,8 +230,85 @@ jobs:
     - name: Archive test results folder
       uses: actions/upload-artifact@v2
       with:
+        name: Results Logs ${{ matrix.node-version }}
```
Comment (newtewt) [inline .github/workflows/build.yml:233]: This seems to be a duplicate job `tests-e2e-disabled`. If it is necessary can you update the name and it's key?

File: tests/conf.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ const baseConfig = {
   suites: {
     // e2e:'e2e/**/*.js',
     e2e: [
+      'e2e/login/login.specs.js',
```
Comment (newtewt) [inline tests/conf.js:23]: These have been moved to the conf-disabled already. Can you remove from here?

```
      'e2e/create-meta-db.js',
      'e2e/login/login.specs.js'
```

File: tests/utils.js [hunk 3]
```diff
@@ -520,7 +520,6 @@ module.exports = {
     captureOnlyFailedSpecs: true,
     reportOnlyFailedSpecs: false,
     showQuickLinks: true,
-    dest: `tests/results/`,
```
Comment (newtewt) [inline tests/utils.js:523]: Is this intentional? It will break the artifact path when run on github.  If it is, then the build.yml needs updated as well.

**Review by newtewt — CHANGES_REQUESTED** [review]
Nice work on moving to the disabled flow. A few questions

**Comment by dianabarsan** [comment]
I've already enabled these tests, without needed changes.

**Pull request closed without merging**